spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -79,6 +79,38 @@ object external {
     new File(path).getAbsoluteFile().toURI()
   }
 
+  // ================ HBase entities ======================
+  val HBASE_NAMESPACE_STRING = "hbase_namespace"
+  val HBASE_TABLE_STRING = "hbase_table"
+  val HBASE_COLUMNFAMILY_STRING = "hbase_column_family"
+  val HBASE_COLUMN_STRING = "hbase_column"
+
+  private def stripNameSpace(tableName: String): String = {
+    tableName.substring(tableName.indexOf(":") + 1)
+  }
+
+  def hbaseTbUniqueAttribute(cluster: String, tableName: String, nameSpace: String) =
+    nameSpace.toLowerCase + '.' + stripNameSpace(tableName.toLowerCase()) + '@' + cluster
+
+  def hbaseTableToEntity(cluster: String, tableName: String, nameSpace: String): Seq[AtlasEntity] = {
+    val hbaseEntity = new AtlasEntity(HBASE_TABLE_STRING)
+    hbaseEntity.setAttribute(AtlasClient.REFERENCEABLE_ATTRIBUTE_NAME,
+      hbaseTbUniqueAttribute(cluster, tableName, nameSpace))
+    hbaseEntity.setAttribute(AtlasClient.NAME, tableName.toLowerCase)
+    hbaseEntity.setAttribute(AtlasConstants.CLUSTER_NAME_ATTRIBUTE, cluster)
+    hbaseEntity.setAttribute("uri", nameSpace.toLowerCase + ":" + tableName.toLowerCase)
+    Seq(hbaseEntity)
+  }
+
+  // TODO: we will make HBase entities in column level
```

Comment:
If this is a work item, could you make an issue on https://github.com/hortonworks-spark/spark-atlas-connector/issues/new and add issue id here?